packages/interfaces/src/light_push.ts

```diff
@@ -4,11 +4,13 @@ import type {
   ProtocolOptions,
   SendResult,
 } from "./protocols.js";
+import { ISender } from "./sender.js";
 
-export interface ILightPush extends PointToPointProtocol {
-  push: (
-    encoder: IEncoder,
-    message: IMessage,
-    opts?: ProtocolOptions
-  ) => Promise<SendResult>;
-}
+export type ILightPush = ISender &
+  PointToPointProtocol & {
+    push: (
```

Comment:
I think it should be simplified to `sender`
@fryorcraken any reasons not to?

Reply:
I agree that part of the sender/receiver work, it'll be a good opportunity to modify the existing API so they can all have the same function signature (ie, lightpush push and relay send, filter subscribe and relay add observer)

Reply:
Renamed to `send`.